index.js

```diff
@@ -10,6 +10,17 @@ var CSS_ESCAPED_TAB = '\\9';
 
 function increaseSpecifityOfRule(rule, opts) {
 	rule.selectors = rule.selectors.map(function(selector) {
+		// Ignore a selector if it is in the ignore list
+		if (opts.ignoreList.length > 0) {
+			var isInIgnoreList = opts.ignoreList.some(function(item) {
+				var regEx = new RegExp(item.replace('.', '\\.') + '\\b', 'gi');
```

Comment:
We should precompute these regexes outside of the `rule.selectors.map` loop.

---

Some small nits

 - `opts.ignoreList.some(function(ignoreItem) {`
 - `regEx -> `ingoreRe`

---

We should note in the docs that this doesn't handle complex ingore selectors.

ex.

 - Ignore `.foo.bar`
 - Selector: `.foo.QUX.bar` would still increase specificity even though it is technically under the `.foo.bar` scope